api-test/src/main/resources/pms/ABISPartnerV3/CreateABISPartner/CreateABISPartner.yml

```
@@ -0,0 +1,27 @@
+CreateABISPartner:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)